internal/node/node_test.go

```diff
@@ -220,3 +269,64 @@ var _ = Describe("FindNodeCondition", func() {
 		Expect(conditionFound).To(BeNil())
 	})
 })
+var kernelModuleReadyNodeLabel = utils.GetKernelModuleReadyNodeLabel("loaded-ns", "loaded-n")
+
+var _ = Describe("addLabels", func() {
+	var (
+		n    Node
+		node v1.Node
+	)
+
+	BeforeEach(func() {
+		n = NewNode(nil)
+		node = v1.Node{}
+	})
+
+	It("Should add labels", func() {
+		labels := []string{kernelModuleReadyNodeLabel}
+
+		Expect(node.Labels).ToNot(HaveKey(kernelModuleReadyNodeLabel))
+		n.addLabels(&node, labels)
+		Expect(node.Labels).To(HaveKey(kernelModuleReadyNodeLabel))
+	})
+	It("Should not add labels", func() {
+		var labels []string
+
+		Expect(node.Labels).ToNot(HaveKey(kernelModuleReadyNodeLabel))
+		n.addLabels(&node, labels)
+		Expect(node.Labels).ToNot(HaveKey(kernelModuleReadyNodeLabel))
+	})
+})
```

Comment:
We can remove this test - it is not really testing anything different than the previous spec.

Reply:
@ybettan  why not? in theory `kernelModuleReadyNodeLabel` can be added no matter what in `addLabels`, so dont we need to test in both directions? (one with kernelModuleReadyNodeLabel and one without kernelModuleReadyNodeLabel)

Reply:
In the previous test, you start with an empty node, add a label and check if the label was added.
In this test, you start with an empty node, call `addLabels` with an empty list and check that nothing happened.
I am not sure what added value this test have over the previous one.

Reply:
test is removed